WcaOnRails/app/views/panel/index.html.erb

```diff
@@ -34,6 +34,24 @@
       <li>
         <%= link_to "Equipment Funding Form", "https://docs.google.com/forms/d/e/1FAIpQLSeglibvXYG6Z2CzIPbxu2LTpAuXkJPK-vi4GuOnBFx1756ufA/viewform" %>
       </li>
+      <li>
+        <%= link_to "Groupifier", "https://groupifier.jonatanklosko.com/" %>
+      </li>
+      <li>
+        <%= link_to "WCA Scrambles Matcher", "https://scrambles-matcher.worldcubeassociation.org/" %>
+      </li>
+      <li>
+        <%= link_to "Results Workbook Template", "https://www.worldcubeassociation.org/files/results.xls" %>
+      </li>
```

Comment:
Wouldn't it be better to have a single link to the [tools page](https://www.worldcubeassociation.org/score-tools)? All three of these are listed there.

Reply:
oh wow. I had no idea about that link :P (Surprisingly all my friends do know about it).

Umm, now knowing about that page, I dont even think a link to it is needed :P. That would eliminate all the changes except the google groups, which IDK how useful those links are. I'll remove the tools and if we want to add the google groups we can otherwise we can close this PR.

Reply:
Additional reference never hurts, let's go with the google group links =)